### PR TITLE
valet: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.es/osx/valet.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`valet park`
+
+- Ver las rutas 'aparcadas':
+
+`valet paths`
+
+- Sirve un único sitio en lugar de un directorio completo:
+
+`valet link nombre-aplicacion`
```
Comment (kbdharun) [inline pages.es/osx/valet.md:20]:
```suggestion
`valet link {{nombre_aplicacion}}`
```

**Review by kbdharun — COMMENTED** [review]
From #10293

**Comment by kant** [comment]
@kbdharun Updated!

**Comment by kant** [comment]
I clicked in the wrong button, I will try to reopen this pull request again...

**Comment by kant** [comment]
[Codespell](https://github.com/tldr-pages/tldr/actions/workflows/codespell.yml)

**Pull request merged**